### PR TITLE
Replace comment placeholder with social media links

### PR DESCRIPTION
**Pull request by contolini**
We're taking an extended break from blog comments.

## Changes

- Text at the bottom of blog posts now links to FB/Twitter.

## Screenshots

b60e1754-235f-11e7-8add-d119bd74317f

## Notes

- See https://[GHE]/CFGOV/platform/issues/779

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

File: cfgov/jinja2/v1/blog/blog_page.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
     {% include 'article.html' %}
     <div class="block">
         <p>
-            Due to technical issues, the commenting feature of our blog is temporarily unavailable. We’re working to bring this functionality back, and look forward to hearing your feedback and comments about the CFPB’s work soon.
+            Join the conversation. Follow CFPB on <a class="icon-link icon-link__external-link" href="https://twitter.com/cfpb"><span class="icon-link_text">Twitter</span> </a> and <a class="icon-link icon-link__external-link" href="https://www.facebook.com/cfpb"><span class="icon-link_text">Facebook</span> </a>.
```
Comment (sebworks) [inline cfgov/jinja2/v1/blog/blog_page.html]: should we be using the `unsigned function`?

Comment (Scotchester) [inline cfgov/jinja2/v1/blog/blog_page.html]: I believe so. [See example in the social media molecule](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_includes/molecules/social-media.html#L74), @contolini.

**Comment by jimmynotjim** [comment]
Such a small change but definitely better than the pervious message.

File: cfgov/jinja2/v1/blog/blog_page.html (hunk 1 above)
Comment (contolini) [inline cfgov/jinja2/v1/blog/blog_page.html]: 👍

**Pull request merged**